### PR TITLE
Ames: Initial commit.

### PR DESCRIPTION
**Pull request by alaczek**
This is the initial commit of Ames - a super simple Blockbase child theme featuring a gallery homepage design. Everything else will fall back to the parent theme.

This is an updated version of Antin, which will be retired when Ames launches.

### CODE REVIEW

**Comment by pbking** [comment]
Because only two colors are defined in the palette the missing colors (primary, secondary, tertiary) need to be mapped to the colors that are available.  Using Skatepark as an example I've[ updated those palette mappings](https://github.com/Automattic/themes/pull/5152/commits/0f577d3f065ac956761d75b9db587f5969fcf0d7).

**Review by pbking — APPROVED** [review]
Approved, but due to a lack of index.html template file requires [this fix ](https://github.com/WordPress/gutenberg/pull/36910)to be in place before it can ship.

File: ames/block-template-parts/footer.html [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<!-- wp:group {"style":{"spacing":{"padding":{"top":"20px";"bottom":"20px"}}}} -->
```
Comment (scruffian) [inline ames/block-template-parts/footer.html]: This footer looks very similar to the one that ships in Blockbase. Why do we need a different one?

Comment (alaczek) [reply ames/block-template-parts/footer.html]: Yeah, looks like it's not needed. I'll remove it.

**Comment by pbking** [comment]
presence of `index.html` template fixes the Gutenberg problem.  Approving for release.

**Pull request merged**